html/pfappserver/lib/pfappserver/I18N/i_default.po

```diff
@@ -6996,44 +6996,44 @@ msgstr "pending"
 msgid "pf::pki_provider::packetfence_pki"
 msgstr "PacketFence PKI"
 
-# pf::filter (Portal Profile Filters)
-msgid "pf::profile::filter::connection_type"
+# pf::condition::profile (Portal Profile Filters)
+msgid "pf::condition::profile::connection_type"
```

Comment:
This and all the ones below have changed and are modules anymore.

This should change